app/services/add_ons/create_service.rb

```diff
@@ -2,7 +2,12 @@
 
 module AddOns
   class CreateService < BaseService
-    def create(**args)
+    def initialize(args)
+      @args = args
+      super
```

Comment:
Hmm just realised that we mostly call just `super` which actually pass current attributes to the parent constructor. Shouldn't we use `super()` here?

Reply:
Well, actually the base service initializer is
```ruby
def initialize(current_user = nil)
    @result = Result.new
    @source = CurrentContext&.source
    result.user = current_user.is_a?(User) ? current_user : nil
  end
```

So it does not really matter, `args` will be assigned to the `current_user` argument and then ignored... The main goal in the end, is actually to get rid of this `current_user` params as it is some old legacy logic that should not be there anymore.